driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/syncadapter/SyncMongoClient.java

```diff
@@ -57,6 +57,22 @@ public static void enableSleepAfterCursorOpen(final long sleepMS) {
         sleepAfterCursorOpenMS = sleepMS;
     }
 
+    /**
+     * Unfortunately this is the only way to wait for error logic to complete, since it's asynchronous.
+     * This is inherently racy but there are not any other good options. Tests which require cursor error handling to complete before
+     * execution of the next operation can set this to a positive value.  A value of 256 ms has been shown to work well. The default
+     * value is 0.
+     */
+    public static void enableSleepAfterCursorError(final long sleepMS) {
```

Comment:
Used to fix races for killCursor events that are sent during cursor errors. If no sleep then control returns to the user too quickly and the test misses killcursor events.